recipe/meta.yaml

```diff
@@ -31,9 +31,9 @@ test:
         - tests/helloworld.cxx
         - tests/helloworld.f
         - tests/helloworld.f90
-    commands:
-        - conda inspect objects openmpi  # [osx]
-        - conda inspect linkages openmpi  # [not win]
+#   commands:
+#       - conda inspect objects openmpi  # [osx]
```

Comment:
Did you verify that these fail? I believe it's a specific bug in mpich that causes O_NONBLOCK problems.

Reply:
No, sorry, you are right. Do you think I should revert these changes?

Reply:
Never mind. I'm going to strip this commit.